pkg/util/validation/validation.go

```diff
@@ -121,6 +121,23 @@ func IsDNS1123Subdomain(value string) []string {
 	return errs
 }
 
+const wildcardDNF1123SubdomainFmt = "\\*\\." + dns1123SubdomainFmt
```

Comment:
please add a comment referencing the rfc where this is defined, in case someone refactors it to another file